app/src/main/java/com/kickstarter/ui/fragments/BackingFragment.kt

```diff
@@ -54,10 +54,10 @@ class BackingFragment: BaseFragment<BackingFragmentViewModel.ViewModel>()  {
                 .compose(Transformers.observeForUI())
                 .subscribe { setCardLastFourText(it) }
 
-        this.viewModel.outputs.cardIsGone()
+        this.viewModel.outputs.paymentMethodIsGone()
```

Comment:
What does "Gone" mean in this context? 😅

Reply:
It means that the row is not visible to the user. `GONE` is a visibility flag of a `View`. This line in the description:
> The payment method section is hidden if the backing.paymentSource is not Google Pay, Apple Pay or a credit card.

So basically it'd hide for bank accounts.